README.md

```diff
@@ -475,15 +464,12 @@ The resulting `http.RoundTripper` implements all of the smart functionality of t
 reverse proxy but is exposed as a component that can be embedded in code and used
 anywhere an HTTP client would otherwise be used.
 
-<a id="markdown-contributing" name="contributing"></a>
 ## Contributing
 
-<a id="markdown-license" name="license"></a>
```

Comment:
question: We get links but is the `HREF` the same as what we link to in the table of contents? Is there a function that defines what the anchor name is from a given header since they may contain non-URL safe characters?

Reply:
github automatically converts them:
so in this PR I can go to links with spaces in the headline
https://github.com/asecurityteam/transportd/tree/update_read_me#runtime-settings

it swapped the space with the dash

Reply:
another example:
https://github.com/asecurityteam/transportd/tree/update_read_me#transportd---http-middleware-as-a-service

Reply:
Im guessing bitbucket doesn't do this? is this why we have a habit of embedding our own links? I've seen this in a few other atlassian README's so Im unsure if this is some kind of convention we're used to. Most of the github projects Ive coded on just rely on the automatic links.

Reply:
It's two parts. One is that, yes, it makes it consistently work across hosting providers which enables us to use a single process for adding permalinks and tables of content. The other is that we automated the table of content and anchor generation with tooling (notably https://marketplace.visualstudio.com/items?itemName=AlanWalk.markdown-toc). It makes it so you can add new sections, run the tool, and have an updated markdown file.

Reply:
I see, it appears github has spoiled me, didn't realize this wasn't standard in bitbucket/stash